torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2292,6 +2292,17 @@ def sample_inputs_leaky_relu(op_info, device, dtype, requires_grad):
                requires_grad=requires_grad)) for _ in range(1, N)]
     return tensors
 
+def sample_inputs_avgpool2d(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    cases = (((1, 3, 9, 9), (3, 3), (1, 1), (1, ), True, False, 2),
```

Comment:
Can you add a comment describing what each element represents please

Reply:
Done, thanks for the suggestion @heitorschueroff!